[25.05] immich: 2.0.1 -> 2.2.2

diogotcorreia opened the pull request:
Backport of:
- https://github.com/NixOS/nixpkgs/pull/452365
- https://github.com/NixOS/nixpkgs/pull/457059
- https://github.com/NixOS/nixpkgs/pull/458160

Additionally, update vips from 8.17.1 to 8.17.3, and add `python3Packages.rapidocr`. Did not cherrypick for the latter, since it would require cherry-picking [at least](https://github.com/NixOS/nixpkgs/commits/master/pkgs/development/python-modules/rapidocr-onnxruntime/default.nix) [7 commits](https://github.com/NixOS/nixpkgs/commits/master/pkgs/development/python-modules/rapidocr/default.nix) and would rename another package only to rename it back.

Closes #452385

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [x] [NixOS tests] in [nixos/tests].
  - [x] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [x] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

dotlambda committed:
immich: 2.0.1 -> 2.1.0

Diff: https://github.com/immich-app/immich/compare/v2.0.1...v2.1.0

Changelog: https://github.com/immich-app/immich/releases/tag/v2.1.0
(cherry picked from commit 528bed44bf38168c519cd254a009a45bb76d7030)

dotlambda committed:
immich: 2.1.0 -> 2.2.1

Diff: https://github.com/immich-app/immich/compare/v2.1.0...v2.2.1

Changelog: https://github.com/immich-app/immich/releases/tag/v2.2.0
           https://github.com/immich-app/immich/releases/tag/v2.2.1
(cherry picked from commit 738dcf167d04689293fb6c5caa7af84902a3262f)

diogotcorreia committed:
vips_8_17: 8.17.1 -> 8.17.3

Not-cherry-picked-because: package only exists in stable release

diogotcorreia committed:
python3Packages.rapidocr: init at 3.4.1

Not-cherry-picked-because: commit adding package contains other changes

matteo-pacini commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 457807 -p immich -p nixosTests.immich`
Commit: [`1ff7a815cbf755390d63642db8c3c2e774cd993d`](https://github.com/NixOS/nixpkgs/commit/1ff7a815cbf755390d63642db8c3c2e774cd993d) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/1ff7a815cbf755390d63642db8c3c2e774cd993d..pull/457807/head))
Merge: [`8a8c55d3c7b0ee745b4f6f8a56f2775e8b714c37`](https://github.com/NixOS/nixpkgs/commit/8a8c55d3c7b0ee745b4f6f8a56f2775e8b714c37)

Logs: https://github.com/matteo-pacini/nixpkgs-review-gha/actions/runs/19014934708


---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 test built:</summary>
  <ul>
    <li>nixosTests.immich</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>immich</li>
  </ul>
</details>

matteo-pacini reviewed (approved):
Approved automatically following the successful run of `nixpkgs-review`.

dotlambda committed:
immich: 2.2.1 -> 2.2.2

Diff: https://github.com/immich-app/immich/compare/v2.2.1...v2.2.2

Changelog: https://github.com/immich-app/immich/releases/tag/v2.2.2
(cherry picked from commit 04d40bd838dd8c191c4f95f93d92700e933604d6)

matteo-pacini commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 457807 -p immich -p nixosTests.immich`
Commit: [`2a191118453dc907deb810fc39679d3f8f61f76c`](https://github.com/NixOS/nixpkgs/commit/2a191118453dc907deb810fc39679d3f8f61f76c) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/2a191118453dc907deb810fc39679d3f8f61f76c..pull/457807/head))
Merge: [`5cf74c9eae5d39a46c3cc0dd1e8687d27d5a6eeb`](https://github.com/NixOS/nixpkgs/commit/5cf74c9eae5d39a46c3cc0dd1e8687d27d5a6eeb)

Logs: https://github.com/matteo-pacini/nixpkgs-review-gha/actions/runs/19079205848


---
### `x86_64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>nixosTests.immich</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>immich</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>nixosTests.immich</summary>
<pre>machine # ls: cannot access &#x27;/var/lib/immich/backups/*.sql.gz&#x27;: No such file or directory
machine # ls: cannot access &#x27;/var/lib/immich/backups/*.sql.gz&#x27;: No such file or directory
machine # ls: cannot access &#x27;/var/lib/immich/backups/*.sql.gz&#x27;: No such file or directory
machine # ls: cannot access &#x27;/var/lib/immich/backups/*.sql.gz&#x27;: No such file or directory
machine # ls: cannot access &#x27;/var/lib/immich/backups/*.sql.gz&#x27;: No such file or directory
machine # ls: cannot access &#x27;/var/lib/immich/backups/*.sql.gz&#x27;: No such file or directory
machine # ls: cannot access &#x27;/var/lib/immich/backups/*.sql.gz&#x27;: No such file or directory
machine # ls: cannot access &#x27;/var/lib/immich/backups/*.sql.gz&#x27;: No such file or directory
machine # ls: cannot access &#x27;/var/lib/immich/backups/*.sql.gz&#x27;: No such file or directory
machine # ls: cannot access &#x27;/var/lib/immich/backups/*.sql.gz&#x27;: No such file or directory
!!! Traceback (most recent call last):
!!!   File &quot;&lt;string&gt;&quot;, line 33, in &lt;module&gt;
!!!     machine.wait_until_succeeds(&quot;ls /var/lib/immich/backups/*.sql.gz&quot;)
!!! 
!!! RequestedAssertionFailed: action timed out after 900 tries with one-second pause in-between
cleanup
kill machine (pid 9)
qemu-system-x86_64: terminating on signal 15 from pid 6 (/nix/store/jd20rkmqmkfkcvk2wl2lmzz7acq4svlr-python3-3.12.12/bin/python3.12)
kill vlan (pid 7)
(finished: cleanup, in 0.01 seconds)</pre>
</details>
</details>
